Makefile

```diff
@@ -1,4 +1,4 @@
-SEMANTIC_VER=0.2.1+
+SEMANTIC_VER=0.2.2+
```

Comment:
Sorry, scratch my last comment... this should be v0.3.1, and the comment in CHANGELOG.md to match. My apologies for the round-about.